packages/common-ui-web/src/components/VehicleDynamicWireframe.tsx

```diff
@@ -0,0 +1,75 @@
+import { useMonkTheme } from '@monkvision/common';
+import { partSelectionWireframes } from '@monkvision/sights';
+import {
+  PartSelectionOrientation,
+  VehiclePart,
+  VehicleType,
+  VehicleTypeMapVehicleModel,
+} from '@monkvision/types';
+import { SVGProps } from 'react';
+import { DynamicSVG, DynamicSVGCustomizationFunctions } from './DynamicSVG';
+
+export interface VehicleDynamicWireframeProps {
+  vehicleType: VehicleType;
+  orientation?: PartSelectionOrientation;
+  onClickPart?: (parts: VehiclePart) => void;
+  getPartAttributes?: (part: VehiclePart) => SVGProps<SVGElement>;
+}
+
+const validElement = (element: SVGElement): boolean =>
+  element.id !== '' && element.classList.contains('car-part');
```

Comment:
Switch to normal `function` declaration. Also, rename function to `isCarPartElement`